core/src/gl/shaderProgram.cpp

```diff
@@ -223,28 +223,7 @@ GLuint ShaderProgram::makeCompiledShader(const std::string& _src, GLenum _type)
             infoLog.resize(infoLength);
 
             GL_CHECK(glGetShaderInfoLog(shader, infoLength, NULL, static_cast<GLchar*>(&infoLog[0])));
-            LOGE("Shader compilation failed %s", m_description.c_str());
-
-            std::stringstream ss(source);
-            std::string item;
-            std::vector<std::string> sourceLines;
-            while (std::getline(ss, item, '\n')) { sourceLines.push_back(item); }
-
-            // Print errors with context
-            std::string line;
-            ss.str(infoLog);
-            while (std::getline(ss, line)) {
-                if (line.length() < 2) { continue; }
-
-                int lineNum = 0;
-                sscanf(line.c_str(), "%*d(%d)", &lineNum);
-                logMsg("\nError on line %d: %s\n", lineNum, line.c_str());
-
```

Comment:
@blair1618 it seems one major issue was in the use of stringstream.str(infoLog) instead of creating a new stringstream. Otherwise each line of the infoLog must have been printed here - in any case.